examples/with-tailwindcss/postcss.config.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   plugins: [
+    'postcss-import',
```

Comment:
This plugin should not be included by default. Let's drop it from here and `package.json`.

Reply:
`@import` is supported oob in Next.js?